### PR TITLE
fix(Scripts/ZulGurub): Fixed Jindo's Brain Wash Totem.

### LINKED ISSUE #11558: [ZG] Jin´do the Hexxer - Brain Wash Totem - Player does not get mindcontrolled
### Current Behaviour

Brain Wash Totem gets put down and nothing happens

### Expected Blizzlike Behaviour

The player should be affected by mind control that cannot be removed unless the totem is destroyed

### Source

https://wowwiki-archive.fandom.com/wiki/Jin%27do_the_Hexxer?oldid=2417056

### Steps to reproduce the problem

1. Go to ZG  - Jin'do with 2 chars
2. Wait for brain wash totem
3. nothing happens

### Extra Notes

_No response_

### AC rev. hash/commit

59e45c251eb53abb0ed4b37a57254cbdf415e69a
### Operating system

Manjaro x64

### Custom changes or Modules

mod-eluna
mod-progression-system

### PR DESCRIPTION
**Pull request by UltraNix**
Fixes #11558

## Issues Addressed:

- Closes #11558

## Tests Performed:

- Not tested.

## How to Test the Changes:

Go to ZG - Jin'do with 2 chars
Wait for brain wash totem
Totem charm random player

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by IntelligentQuantum — APPROVED** [review]

**Comment by Maelthyrr** [comment]
@UltraNix it's doesn't work. Totem spawn but do nothing.

**Comment by UltraNix** [comment]
@Maelthyrr Have you tested that with at least 2 people?

**Comment by UltraNix** [comment]
@Maelthyrr Should be okay now.

**Comment by Maelthyrr** [comment]
@UltraNix it's work, just one minor problem. When the totem is destroy, the player don't lost the mind control.

**Comment by UltraNix** [comment]
@Maelthyrr Not related to this PR. Might be a general problem.

File: src/server/scripts/EasternKingdoms/ZulGurub/zulgurub.h [hunk 1]
```diff
@@ -57,9 +57,10 @@ enum CreatureIds
     NPC_MANDOKIR            = 11382, // Mandokir Event
     NPC_OHGAN               = 14988, // Mandokir Event
     NPC_VILEBRANCH_SPEAKER  = 11391, // Mandokir Event
-    NPC_CHAINED_SPIRIT      = 15117, // Mandokir Event
+    NPC_CHAINED_SPIRT       = 15117, // Mandokir Event
```
Comment (Nefertumm) [inline src/server/scripts/EasternKingdoms/ZulGurub/zulgurub.h:60]:
```suggestion
    NPC_CHAINED_SPIRIT      = 15117, // Mandokir Event
```

**Pull request merged**